### PR TITLE
New charater

### PR DESCRIPTION
**Pull request by MashdorDev**
OK I have finished doing #3

### CODE REVIEW

**Review by rylerbleau — CHANGES_REQUESTED** [review]
Make it better please

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # unbread
+New Charater Finish!
```
Comment (rylerbleau) [inline README.md:2]: improper punctuation

**Review by rylerbleau — CHANGES_REQUESTED** [review]

**Comment by MashdorDev** [comment]
hey @rylerbleau, can you check it again and merge it if it's okay?

**Review by rylerbleau — APPROVED** [review]
fine

**Pull request merged**